internal/pkg/agent/application/info/state_windows.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build windows
+
+package info
+
+import (
+	"github.com/elastic/elastic-agent/pkg/utils"
+)
+
+func fixInstallMarkerPermissions(markerFilePath string, ownership utils.FileOwner) error {
+	// TODO(blakerouse): Fix the market permissions on Windows.
```

Comment:
missed or future work? can we link an issue if so?

Reply:
don't really have a public issue, so I didn't link it

I am working on Windows next, this will not live long.